core/server/models/base/index.js

```diff
@@ -19,10 +19,12 @@ var _          = require('lodash'),
     validation = require('../../data/validation'),
     plugins    = require('../plugins'),
     i18n       = require('../../i18n'),
-
     ghostBookshelf,
     proto;
 
+// event connector
+require('./when');
```

Comment:
Can you explain reasoning for name change + moving this code? In terms of simple readability it doesn't make any more sense to me than what was there before - so interested to understand.